test/integration/tasks.test.js

```diff
@@ -370,8 +377,34 @@ describe("Tasks", function () {
           expect(res).to.have.status(401);
           expect(res.body.message).to.be.equal("Unauthenticated User");
           return done();
-        })
-        .catch(done());
+        });
+    });
+  });
+
+  describe("GET /tasks/overdue", function () {
+    it("Should return all the overdue Tasks", async function () {
+      await tasks.updateTask(tasksData[0]);
+      await tasks.updateTask(tasksData[1]);
+      const res = await chai.request(app).get("/tasks/overdue").set("cookie", `${cookieName}=${superUserJwt}`);
+
+      expect(res).to.have.status(200);
+      expect(res.body.newAvailableTasks).to.be.a("array");
+      res.body.newAvailableTasks.forEach((task) => {
+        const { status, startedOn, endsOn, assignee } = task.unassignedTask;
+        expect(status).to.equal("AVAILABLE");
+        expect(assignee).to.equal(null);
+        expect(startedOn).to.equal(null);
+        expect(endsOn).to.equal(null);
+      });
+    });
+
+    it("Should return [] if no overdue task", async function () {
+      await tasks.updateTask(tasksData[2]);
```

Comment:
What in the case if the logged in user is not a superUser? `${cookieName}=${jwt}` ?